translations/subsurface_bg_BG.ts

```diff
@@ -6623,39 +6623,6 @@ If your device uses Bluetooth, do the same preparations as for a logbook downloa
         <translation>Облаков слад (нужен е ПИН)</translation>
     </message>
 </context>
-<context>
```

Comment:
These .ts files are automatically generated by the translation platform that we are using (transifex.com). This means:
- your changes will be overwritten the next time we update the translations files;
- making manual changes has the potential to result in conflicts and problems with the integration with Transifex.

For these reasons please remove these changes from the pull request.